Source/carthage/Update.swift

```diff
@@ -65,15 +65,18 @@ public struct UpdateCommand: CommandProtocol {
 			let buildDescription = "skip the building of dependencies after updating\n(ignored if --no-checkout option is present)"
 
 			let dependenciesUsage = "the dependency names to update, checkout and build"
+			let defaultValue: String? = nil
```

Comment:
@DavidBrunow — as you're probably already aware (but hopefully positively expecting), the merging of https://github.com/Carthage/Carthage/pull/2859 into `master` will cause merge conflicts with this hunk.

And a way over due: major thanks for working on this 🙂👍